rotkehlchen/exchanges/kraken.py

```diff
@@ -295,6 +296,7 @@ def __init__(
         self.call_counter = 0
         self.last_query_ts = 0
         self.history_events_db = DBHistoryEvents(self.db)
+        self.supports_history_events = True
```

Comment:
What is this? Why add a new variable? Just implement or not the function you added